### ISSUE TITLE
Check for defaultTestSession on startup

### ISSUE THREAD

**Issue by iheos** [usnistgov/iheos-toolkit2#422]
There is now a configuration property Default_Test_Session which, if missing in multi-user mode on startup will prevent a user from entering an existing test session id and moving to it.  There is no error message, just no response on the screen.

1. If in single user mode - nothing new - this property is ignored (I think this is right - needs to be verified)
2. If in multi user mode - check that this property exists.  If not display error message on UI AND in server log.

Basically Default_Test_Session is now a required parameter in multiuser mode.
